src/modules/dashboard/components/CoinMachine/CoinMachine.tsx

```diff
@@ -90,9 +87,12 @@ const CoinMachine = ({
     variables: { colonyAddress },
     fetchPolicy: 'network-only',
   });
+  const saleStarted = !bigNumberify(
```

Comment:
Just a nitpick but I prefer `hasSaleStarted` style (kinda more grammatically correct & sound more complete), but as you prefer since it's your code.

Reply:
Sure, sounds good